src/ODEs/ODETools/RungeKutta.jl

```diff
@@ -158,15 +159,14 @@ A(t,uf,∂uf/∂t) = ∂uf/∂t - ∑_{i<=s} b_i * f(ui,ti) = 0
 Uses the vector b as auxiliar variable to store the residual of the update ODE
 operator (e.g. identity or mass matrix), then adds the corresponding contribution from earlier stages.
 ```math
-b = [∂u/∂t]
-b - ∑_{i<=s} bi * f(ui,ti) = 0
+b = M(uf,tf)[∂u/∂t]
```

Comment:
What is uf here?
I don't understand this description of the problem
It does not look like RK

Reply:
clean version does not edit RungeKutta.jl